docs/achtergronddocumentatie/artefacten/0992.md

```diff
@@ -13,30 +13,17 @@ Deze case beschrijft de meest eenvoudige casus. Een klant belt het KCC met een v
 ## 1. Medewerker beantwoordt inkomende oproep
 
 - Medewerker is reeds als actor bekend en ingelogd in de frontend applicatie.
-- In de frontend applicatie voor het KCC wordt een nieuw `klantcontact` gemaakt.
 - De klant noemt zijn/haar naam en stelt een vraag.
-```
-- Wordt hier geen `Betrokkene bij klantcontact` aangemaakt om de naam vast te leggen ?
-```
+- De relevante gegevens die tussen de medewerker en klant worden uitgewisseld worden (tijdelijk) in de frontend applicatie vastgelegd.
 - De medewerker kan deze vraag direct beantwoorden.
-- Het is niet nodig de identiteit van de klant te kennen. De medewerker doet dan ook geen poging tot authenticatie.
+- Indien de identiteit van de klant niet is vastgesteld en er geen sprake is van een gekende klant wordt er geen klantcontact geregistreerd ([Zie Basisterminologie](../../basisterminologie.md#anoniem-telefoongesprek-leidt-niet-tot-vervolgacties) )
+- Als de identiteit van de klant bekend is (en er dus authenticatie heeft plaatsgevonden) wordt er wel een klantcontact aangemaakt. 
```

Comment:
Ik begrijp dat bij functies een strikt onderscheid wordt gehanteerd, maar hoort bij deze casusbeschrijving niet wel het registreren van een betrokkene, eventueel registratie van een nieuwe partij en het relateren van een partij aan de betrokkene?

Reply:
In de functie F2173 - registreer klantcontact  wordt het al dan niet vastleggen van een betrokkene en/of een partij beschreven. Daar verwijs ik in regel 29 nogmaals heel specifiek naar.

Reply:
Ik trek m'n opmerking terug; het is inderdaad niet zinvol hier nog eens over te doen wat al in F2173 beschreven is.